backup/re_initialise.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#
+# (Re-)initialise a system from a backup that has been dumped into the root tree.
+
+chown mastodon:mastodon /opt/mastodon/mastodon.env
+
+source /opt/mastodon/versions.env
+chmod 0444 /opt/backup/mastodon_db_backup.sql
+
+echo "Importing mastodon database"
+# We run this in the background instead of using --detach so the log output remains visible
+/usr/bin/docker run --rm  \
+                  --shm-size=256mb \
+                  --health-cmd="pg_isready -U postgres" \
+                  --env POSTGRES_HOST_AUTH_METHOD=trust \
+                  --volume=/opt/backup/mastodon_db_backup.sql:/docker-entrypoint-initdb.d/mastodon_db.sql \
+                  --volume=/opt/mastodon/postgres:/var/lib/postgresql/data \
+                  --name=mastodon-db-import \
+                  "${POSTGRES_IMAGE}" &
+
+# 20 minutes max for db import
+/opt/mastodon/wait-for-container.sh mastodon-db-import .State.Health healthy 1200
```

Comment:
What about adding an extra `wait` to be sure that the background process started above is over for sure?

Reply:
The container is only used to import the SQL dump - that happens in the initdb part, which runs before the actual database comes up inside the container. As long as the DB is not up, the health check  `pg_isready -U postgres` will not succeed. In other words, when the container is reported "healthy", we are sure that the import concluded. `wait-for-container.sh` will block until that happened.

Lastly, `docker stop` (below) will block until the import container has shut down.

Reply:
I've added comments in the file on how things are supposed to work :)

Reply:
Ok, thanks for elaborating!